README.md

```diff
@@ -160,8 +160,18 @@ Run `make build-all`.
 ## Try Out the Application
 
 Once your application is deployed, you can try it out by navigating to `https://$DOMAIN`,
-where `$DOMAIN` is the custom domain
-you configured in `env.mk`.
+where `$DOMAIN` is the custom domain you configured in `env.mk`.
+
+### Setup first admin user
+
+After you login at least once to the app, you can use this script to update your account to `admin`.
+Afterwards you can use Users page to manage other accounts. To use this script you will need to follow the
+guide of [Initialize the firebase Admin SDK][] and setup `GOOGLE_APPLICATION_CREDENTIALS` environment variable.
```

Comment:
nit: wording

```suggestion
After you login at least once to the app, you can use this script to make your account an `admin`.
Afterwards you'll be able to use the Users page to manage other accounts. To use this script you will need to  [Initialize the Firebase Admin SDK][] and setup the `GOOGLE_APPLICATION_CREDENTIALS` environment variable.
```

On a side note, it's kind of shame we need to have users go through extra steps to get this to work. I can't think of a better way though :/